src/PFXImportPowershell/EncryptionUtilities/Source/ManagedRSAEncryption.cs

```diff
@@ -569,10 +602,10 @@ private HashAlgorithmName GetHashAlgorithmNameFromPlaintext(string hashAlgorithm
                 throw new CryptographicException(
                     string.Format(
                         "Attempting to find HashAlgorithm for {0} failed, only supported algorithms are {1}, {2}, {3}, {4}",
-                        hashAlgorithm, 
-                        PaddingHashAlgorithmNames.SHA1, 
-                        PaddingHashAlgorithmNames.SHA256, 
-                        PaddingHashAlgorithmNames.SHA384, 
+                        hashAlgorithm,
+                        PaddingHashAlgorithmNames.SHA1,
```

Comment:
Is that correct? Didn't we take SHA1 off?
Hmm... in [line 584](https://github.com/microsoft/Intune-Resource-Access/blob/727a996d02243885e27a5bf3e4fdf13619900fac/src/PFXImportPowershell/EncryptionUtilities/Source/ManagedRSAEncryption.cs#L584) we allow it.

Reply:
Yeah. We block it at the higher level. We should get rid of it everywhere though, shouldn't we.